packages/react-utilities/tsconfig.json

```
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "outDir": "dist",
+    "target": "es5",
+    "module": "commonjs",
+    "jsx": "react",
+    "declaration": true,
+    "sourceMap": true,
+    "experimentalDecorators": true,
+    "importHelpers": true,
+    "noUnusedLocals": true,
+    "forceConsistentCasingInFileNames": true,
+    "strict": true,
+    "moduleResolution": "node",
+    "preserveConstEnums": true,
+    "lib": ["es2015", "dom", "es2015.promise"],
+    "typeRoots": ["../../node_modules/@types", "../../typings"],
+    "types": ["jest", "webpack-env", "custom-global"],
+    "paths": {
+      "@fluentui/react-utilities/lib/*": ["./src/*"],
+      "@fluentui/react-utilities": ["./src"]
+    }
```

Since the package doesn't have examples:
```suggestion
    "types": ["jest", "webpack-env", "custom-global"]
```
(we really ought to add an option to create-package about whether the package will have examples, so I don't have to keep making this suggestion)

---

I'm afraid to reset the CI as this is the 3rd attempt, and I don't think having these paths is going to cause confusion or problems, but happy to merge in a separate pr later when I'm in the area.